linkatos/message.py

```diff
@@ -1,7 +1,11 @@
 import re
+import os
 
 link_re = re.compile("https?://\S+(\s|$)")
 yes_re = re.compile("(\s|^)(Yes|YES|yes)(\s|[,.]|$)")
+no_re = re.compile("(\s|^)(No|NO|no)(\s|[,.]|$)")
+BOT_ID = os.environ.get("BOT_ID")
```

Comment:
This shouldn't be here. You don't want environment variables scattered in your program. Instead, keep the env var in `linkatos.py` and pass the value to the function that needs it.